gensim/models/keyedvectors.py

```diff
@@ -1052,15 +1178,12 @@ def _log_evaluate_word_analogies(section):
             logger.info("%s: %.1f%% (%i/%i)", section['section'], 100.0 * score, correct, correct + incorrect)
             return score
 
-    def evaluate_word_analogies(self, analogies, restrict_vocab=300000, case_insensitive=True, dummy4unknown=False):
+    def evaluate_word_analogies(self, analogies, restrict_vocab=300000, case_insensitive=True, dummy4unknown=False, topk=1, method='3CosAdd'):
         """Compute performance of the model on an analogy test set.
-
```

Comment:
-1: easier to read with the comment blocks separated by blank lines.

Reply:
Fixed